docs/content/v0.20/architecture.md

```diff
@@ -229,20 +225,39 @@ pattern has been extended to provide the following features:
     into/from the OCM mailbox namespaces.
 
 Currently there are two controllers in the KubeStellar controller
-manager: the placement controller and the status controllor.
+manager: the binding controller and the status controllor.
+
+### Planned changes to the above:
+
+as part of the decoupling of transport from binding as described above, the following changes are included:
+- Binding Controller - one client connected to the WDS space and 
+  (or more in the future) to connect to one or more ITS shards.
+  - The WDS-connected client is used to start the dynamic
+    informers/listers for most API resources in the WDS.
+  - The OCM-connected client is used to start informers/listers for OCM
+    ManagedClusters. This is a temporary state until cluster inventory abstraction is implemented and decoupled from OCM (and then this client should be removed and we would need to use client to inventory space). 
+- Transport controller - one client connected to the WDS space 
+  and one client (or more in the future) to connect to one or more ITS shards.
+  - The OCM-connected client is used to copy/update/remove the wrapped objects
+    into/from the OCM mailbox namespaces.
+
+so after the described change, we will have three controllers in the KubeStellar controller
+manager: the binding controller, the transport controller and the status controllor.
 
-### Placement Controller
+### Binding Controller (a.k.a. Placement Controller)
 
-The Placement controller is responsible for watching workload and 
-placement objects, and wrapping and delivering objects to the ITS
-based on placement policies.
+The Binding controller is responsible for watching workload objects and 
+`BindingPolicy` objects, and wrapping and delivering objects to the ITS
+based on binding policies. as a step towards the planned change that is describe in this document, Binding Controller also creates for each `BindingPolicy` a matching `Binding` object in WDS. a `Binding` object is mapped 1:1 to a `BindingPolicy` object and contains the concrete list of workload objects and concrete list of destinations that were selected by the policy and
+should get all the workload objects.
+Until the planned change is completed, the `Binding` objects are not used. 
```

Comment:
'are not used' is very vague. It could even mean that the concept of binding object is not used in the code. This is not what we mean here. I would say it a little clearer, perhaps something like "nothing besides this controller touches the `Binding` objects".

Reply:
fixed